docs/api.md

```diff
@@ -3040,6 +3040,270 @@ name,label,first_name,last_name,age,favorite_color
+
++ Parameters
+    + projectId: `16` (number, required) - The numeric ID of the Project
+    + name: `people` (string, required) - Name of the Dataset
+    + uuid: `54a405a0-53ce-4748-9788-d23a30cc3afa` (string, required) - UUID of the Entity
+
++ Response 200 (application/json)
+    + Response 200
+    + Attributes (array[array[Entity Diff Value]])
+
+    + Body
+
+            {
+              2: [
+                {
+                  "new": "John",
+                  "old": "Dana",
+                  "propertyName": "firstName"
+                },
+                {
+                  "new": "Doe",
+                  "old": "Roe",
+                  "propertyName": "lastName"
+                },
+                {
+                  "new": "John Doe",
+                  "old": "Jane Roe",
+                  "propertyName": "label"
+                }
+              ]
+            }
+
++ Response 403 (application/json)
+    + Attributes (Error 403)
+
+## Entity Audit Log [GET /projects/{projectId}/datasets/{name}/entities/{uuid}/audits]
+
+Returns [Server Audit Logs](/reference/system-endpoints/server-audit-logs) relating to an Entity. They will be returned most recent first.
+
+This endpoint supports retrieving extended metadata; provide a header `X-Extended-Metadata: true` to additionally expand the `actorId` into full `actor` details, and `acteeId` into full `actee` details. The `actor` will always be an Actor, and the `actee` will be the Dataset this Entity is a part of.
+
++ Parameters
+    + projectId: `16` (number, required) - The numeric ID of the Project
+    + name: `people` (string, required) - Name of the Dataset
+    + uuid: `54a405a0-53ce-4748-9788-d23a30cc3afa` (string, required) - UUID of the Entity
+
++ Response 200 (application/json)
+    + Attributes (array[Audit])
+
++ Response 200 (application/json; extended)
+    + Attributes (array[Extended Audit])
+
++ Response 403 (application/json)
+    + Attributes (Error 403)
+
+## Creating an Entity [POST /projects/{projectId}/datasets/{name}/entities]
+
+Creates an Entity in the Dataset. Request body takes the json representation of the entity. It should have `uuid` and `label` property in addition to the user-defined properties of the Dataset. For e.g.
+```
+{
+    "uuid": "54a405a0-53ce-4748-9788-d23a30cc3afa",
+    "label": "John Doe",
+    "firstName": "John",
+    "lastName": "Doe",
+    "city": "New York",
+    ...
+}
+```
+
+You can provide header `X-Client-Id` to identify the client/transaction of the consumer of the API. The value is logged and can be retrieved from the [Getting Entity Details](#reference/entities/getting-entity-details) and [Listing Entity Versions](#reference/entities/listing-entity-versions) APIs
```

Comment:
Is this something we want to support long-term? If we don't need to include it in .3, then we could wait and see how the entity space continues to develop. If we add API token actors at some point, then maybe this header wouldn't be needed anymore.

Reply:
Concluded in the meeting to use `userAgent` instead